node/src/main/scala/com/wavesplatform/database/LevelDBWriter.scala

```diff
@@ -172,14 +171,18 @@ class LevelDBWriter(writableDB: DB, spendableBalanceChanged: Observer[(Address,
     Map.empty
   )
 
-  private def loadPortfolio(db: ReadOnlyDB, addressId: BigInt) = loadLposPortfolio(db, addressId).copy(
+  private def loadPortfolio(db: ReadOnlyDB, addressId: BigInt, excludeNFT: Boolean) = loadLposPortfolio(db, addressId).copy(
     assets = (for {
-      asset <- db.get(Keys.assetList(addressId))
+      issuedAsset <- db.get(Keys.assetList(addressId))
+      asset <- transactionInfo(issuedAsset.id).collect {
+        case (_, itx: IssueTransaction) if !excludeNFT || !itx.isNFT => IssuedAsset(itx.assetId())
+      }
     } yield asset -> db.fromHistory(Keys.assetBalanceHistory(addressId, asset), Keys.assetBalance(addressId, asset)).getOrElse(0L)).toMap
   )
 
   override protected def loadPortfolio(address: Address): Portfolio = readOnly { db =>
-    addressId(address).fold(Portfolio.empty)(loadPortfolio(db, _))
+    val excludeNFT = activatedFeatures.contains(BlockchainFeatures.ReduceNFTFee.id)
```

Comment:
```suggestion
    val excludeNFT = activatedFeatures.get(BlockchainFeatures.ReduceNFTFee.id).exists(_ <= height)
```
because `(featureId, height)` pairs are added to the map when voting finishes, but before activation height is reached. Please test this!